app/models/planet.py

```diff
@@ -0,0 +1,8 @@
+from app import db
+
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+    color = db.Column(db.String)
```

Comment:
Careful! Using the class as it currently is, we'd be able to make a planet with a `null` name, description, and color. When making classes in the future, consider which columns are required (non-nullable).